### PR TITLE
Automated description field feature for user

### PR DESCRIPTION
**Pull request by ntkathole**
Issue : #4947

### CODE REVIEW

File: tests/foreman/ui/test_user.py [hunk 1]
```diff
@@ -639,6 +659,26 @@ def test_positive_update_email(self):
...
+        :expectedresults: User is updated successfully
+
+        :CaseImportance: Critical
+        """
+        description = gen_string('alpha')
+        new_description = gen_string('alpha')
+        username = gen_string('alpha')
+        with Session(self.browser) as session:
+            make_user(session, username=username, description=description)
+            self.user.update(username, description=new_description)
```
Comment (ldjebran) [inline tests/foreman/ui/test_user.py]: we can use (as create function) the valid_data_list

**Comment by ldjebran** [comment]
Please provide test results

File: tests/foreman/ui/test_user.py [hunk 2]
```diff
@@ -35,6 +35,7 @@
     invalid_names_list,
     invalid_values_list,
     valid_emails_list,
+    valid_data_list,
```
Comment (abalakh) [inline tests/foreman/ui/test_user.py]: nitpick: please sort alphabetically.

File: tests/foreman/ui/test_user.py (hunk 1 above)
Comment (abalakh) [inline tests/foreman/ui/test_user.py]: Yep, as @ldjebran said, use subtest & valid_data_list same way as in create test (you may take a look at similar update tests)

**Comment by ntkathole** [comment]
@ldjebran and @abalakh, thank you for your reviews...I had made some changes and Please find the test results below

'''
pytest tests/foreman/ui/test_user.py::UserTestCase::test_positive_create_with_description
============================= test session starts ==============================
collected 56 items 
2017-07-13 16:53:59 - conftest - DEBUG - Deselect of WONTFIX BZs is disabled in settings

tests/foreman/ui/test_user.py .

========================== 1 passed ===========================
'''
'''
pytest tests/foreman/ui/test_user.py::UserTestCase::test_positive_update_description
============================= test session starts ==============================
collected 56 items 
2017-07-13 17:04:36 - conftest - DEBUG - Deselect of WONTFIX BZs is disabled in settings

tests/foreman/ui/test_user.py .

========================== 1 passed ==========================
'''

File: tests/foreman/ui/test_user.py [hunk 3]
```diff
@@ -639,6 +659,28 @@ def test_positive_update_email(self):
...
+
+        :CaseImportance: Critical
+        """
+        username = gen_string('alpha')
+        description = gen_string('alpha')
+        with Session(self.browser) as session:
+            make_user(session, username=username, description=description)
+        for new_description in valid_data_list():
+            with self.subTest(new_description):
+                with Session(self.browser):
```
Comment (abalakh) [inline tests/foreman/ui/test_user.py]: No need in extra session (another login/logout), just continue using previous one.

**Comment by ntkathole** [comment]
@abalakh thank you for review...updated changes

**Pull request merged**